i,size,mu,method,random,remerge,relative,time,modularity,conductance,jaccard
0,1000,0.4,None,True,False,False,5.754533767700195,0.5618189279627723,0.5998995865402648,0.9404834676298972
1,1000,0.4,None,True,False,False,8.125368356704712,0.5628005182067192,0.6017978997093869,0.941740770681757
1000_0.4_2,ERRORException type : KeyboardInterrupt Exception message : Stack trace : ['File : c:\\Users\\sabam\\OneDrive - mail.tau.ac.il\\Biological Networks\\BioNetworkClustering\\YemeniteStep\\yemeniteStepTesterLinux.py , Line : 85, Func.Name : run_test, Message : write_result(f, i, n, mu, method, rnd, rmrg, rltv, time_, comms, dic, real_comms, G)', 'File : c:\\Users\\sabam\\OneDrive - mail.tau.ac.il\\Biological Networks\\BioNetworkClustering\\YemeniteStep\\yemeniteStepTesterLinux.py , Line : 32, Func.Name : write_result, Message : cunductance = str(evaluation.conductance(G, comms))', 'File : c:\\Users\\sabam\\OneDrive - mail.tau.ac.il\\Biological Networks\\BioNetworkClustering\\YemeniteStep\\evaluation.py , Line : 30, Func.Name : conductance, Message : T=G.nodes()-A', 'File : C:\\Program Files (x86)\\Python38-32\\lib\\_collections_abc.py , Line : 501, Func.Name : __sub__, Message : return self._from_iterable(value for value in self', 'File : C:\\Users\\sabam\\AppData\\Roaming\\Python\\Python38\\site-packages\\networkx\\classes\\reportviews.py , Line : 201, Func.Name : _from_iterable, Message : return set(it)', 'File : C:\\Program Files (x86)\\Python38-32\\lib\\_collections_abc.py , Line : 501, Func.Name : <genexpr>, Message : return self._from_iterable(value for value in self', "File : c:\\Users\\sabam\\.vscode\\extensions\\ms-python.python-2022.8.1\\pythonFiles\\lib\\python\\debugpy\\_vendored\\pydevd\\_pydevd_bundle\\pydevd_trace_dispatch_regular.py , Line : 374, Func.Name : __call__, Message : return None if event == 'call' else NO_FTRACE"]
